src/libsyntax/feature_gate/active.rs

```diff
@@ -529,6 +529,9 @@ declare_features! (
     /// Allows using the `#[register_attr]` attribute.
     (active, register_tool, "1.41.0", Some(66079), None),
 
+    /// Allows the use of `if` and `match` in constants.
+    (active, const_if_match, "1.41.0", Some(49146), None),
```

Comment:
Btw: a useful trick for the future, if you want to avoid rebasing due to this, is to move this before `never_type_fallback`.